src/ListView.vala

```diff
@@ -46,8 +46,16 @@ public class Tasks.ListView : Gtk.Grid {
         settings_button.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
         settings_button.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
+        var placeholder = new Gtk.Label (_("No Reminders"));
+        placeholder.show ();
```

Comment:
It looks way too low visually, would this be awful?

```suggestion
        placeholder.yalign = 0.33;
        placeholder.show ();
```

![Screenshot from 2019-12-18 12 23 30](https://user-images.githubusercontent.com/611168/71116450-6f546d80-2191-11ea-96ee-2a15ec7af68f.png)

Reply:
I think this will resolve itself when there's UI for adding tasks at the bottom of the view. I'd be inclined to hold off on this for now